pkg/reconciler/apis/apibinding/conflict_checker.go

```diff
@@ -53,33 +53,13 @@ func (ncc *conflictChecker) getBoundCRDs(apiBindingToExclude *apisv1alpha1.APIBi
 		if apiBinding.Name == apiBindingToExclude.Name {
 			continue
 		}
-
-		apiExportClusterName, err := getAPIExportClusterName(apiBinding)
-		if err != nil {
-			return err
-		}
-
-		apiExport, err := ncc.getAPIExport(apiExportClusterName, apiBinding.Spec.Reference.Workspace.ExportName)
-		if err != nil {
-			return err
-		}
-
 		boundSchemaUIDs := sets.NewString()
 		for _, boundResource := range apiBinding.Status.BoundResources {
 			boundSchemaUIDs.Insert(boundResource.Schema.UID)
 		}
 
-		for _, schemaName := range apiExport.Spec.LatestResourceSchemas {
-			schema, err := ncc.getAPIResourceSchema(apiExportClusterName, schemaName)
-			if err != nil {
-				return err
-			}
-
-			if !boundSchemaUIDs.Has(string(schema.UID)) {
-				continue
-			}
-
-			crd, err := ncc.getCRD(ShadowWorkspaceName, string(schema.UID))
+		for _, bs := range apiBinding.Status.BoundResources {
+			crd, err := ncc.getCRD(ShadowWorkspaceName, bs.Schema.UID)
```

Comment:
Why more `bs`? `resource` or `boundResource` would work ... we should prefer names that mean something

Reply:
brs? :) but yes, bs is bad here as it usually means bytes.